about.html

```diff
@@ -74,34 +80,34 @@
             <p class="welcome-about">"Hello! Sharing World"</p>
-            <h1 class="topic-about">CREATIVE COMMONS GLOBAL SUMMIT 2015</h1>
-            <p class="main-desc-about">Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum</p>
-            <p class="contact-desc">Please contact us per Email for any further questions about CC Global Summit 2015!</p>
+            <h1 class="topic-about">LARGEST TECH CONFERENCE 2022</h1>
+            <p class="main-desc-about"> Largest Tech Conference 2022 is back on october 4 - 7,  2022. Every year for the last 9 years, the world’s smartest marketers have descended upon New York, to learn what’s new, what’s hot, and what’s actually working RIGHT NOW in digital marketing. Now, it’s your turn.</p>
+            <p class="contact-desc">Please contact us per Email for any further questions about the Largest Tech Conference 2022!</p>
             <a href="#" class="mail-link">
-                <span>ccsummit2015@cckorea.org</span>
+                <span>largesttechcnference@ltckorea.org</span>
                 <i class="fa-solid fa-angle-right"></i>
             </a>
         </section>
 
         <section class="about-2">
-            <h2 class="headings heading-4">Creative Commons Global Summit 2015 Logo</h2>
-            <p class="about-2-desc">Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum</p>
+            <h2 class="headings heading-4">Largest Tech Conference 2022 Logo</h2>
+            <p class="about-2-desc"> Largest Tech Conference 2022 is back on october 4 - 7,  2022. Every year for the last 9 years, the world’s smartest marketers have descended upon New York, to learn what’s new, what’s hot, and what’s actually working RIGHT NOW in digital marketing. Now, it’s your turn.</p>
             <div class="about-img">
-                <img src="img/partner-1.png" class="image">
+                <img alt="Robust-Events" src="img/logo.png" class="image">
```

Comment:
- Currently all your alt tags have the same description, just as the previous reviewer requested  kindly change these and add  appropriate descriptions of images in the alt tags, this makes your website more accessible and helps screen readers read your website easily 👍🏿